x-pack/dockerlogbeat/pipelinemanager/clientLogReader.go

```diff
@@ -97,10 +100,20 @@ func (cl *ClientLogger) publishLoop(reader chan logdriver.LogEntry) {
 			return
 		}
 
+		line := strings.TrimSpace(string(entry.Line))
+
 		cl.client.Publish(beat.Event{
 			Timestamp: time.Unix(0, entry.TimeNano),
 			Fields: common.MapStr{
-				"line": string(entry.Line),
+				"message": line,
+				"container": common.MapStr{
+					"labels": cl.containerMeta.ContainerLabels,
+					"id":     cl.containerMeta.ContainerID,
+					"name":   cl.containerMeta.ContainerName,
+					"image": common.MapStr{
+						"name": cl.containerMeta.ContainerImageName,
+					},
```

Comment:
would it be possible to add unit tests for this?

Reply:
Yah, unit tests are a whole item on  #13990. I figured any meaningful test suite would require a lot of harness code, since most of the codebase is interacting with outside APIs.

Reply:
SGTM :+1: , in general I see many opportunities to do unit testing before going into external APIs